test/integration/listener_lds_integration_test.cc

```diff
@@ -1356,6 +1356,146 @@ TEST_P(ListenerFilterIntegrationTest, UpdateListenerFilterOrder) {
   tcp_client3->close();
 }
 
+#if !defined(__APPLE__) && !defined(WIN32)
```

Comment:
I should add a test for the mac and windows

Reply:
emm...no, `UpdateListenerWithDifferentSocketOptionsWhenReusePortDisabled` is enough, there should be no different to test on explicit set enable_reuse_port as false or test on the default value of enable_reuse_port as false.